shopify-client/src/views/ProductDetails.vue

```diff
@@ -12,7 +12,7 @@
           <b-col sm="6">
             <b-row>
               <b-col sm="12">
-                <b-img v-bind:src="product.image" fluid></b-img>
+                <b-img v-bind:src="'data:image/png;base64,' + product.image" fluid></b-img>
```

Comment:
Doesn't `readAsDataURL` already add the `data:image/png;base64` to the encoded image when we submitted it to the server?

Reply:
No, I striped the data thing because it is not valid Base64.